repos/spack_repo/builtin/packages/fenics_dolfinx/package.py

```diff
@@ -18,6 +18,9 @@ class FenicsDolfinx(CMakePackage):
     license("LGPL-3.0-or-later")
 
     version("main", branch="main", no_cache=True)
+    version(
+        "0.10.0.post1", sha256="43ebfaf89fcb7809b11b94ba3e8ea8e56a99d724a45c51e99e39a35c577c441e"
```

Comment:
Do we know if reflecting post release numbering is correct in Spack? After all, there is no functional difference with 0.10, just packaging tweaks.